gtda/time_series/features.py

```diff
@@ -29,7 +29,7 @@ class PermutationEntropy(BaseEstimator, TransformerMixin):
 
     See also
     --------
-    TakensEmbedding, gtda.diagrams.PersistenceEntropy
+    SingleTakensEmbedding, gtda.diagrams.PersistenceEntropy
```

Comment:
`SlidingWindow` should be there too as it is the main way people are going to get  collections of time-series.

Reply:
Thanks, well spotted!

Reply:
Fixed (and mentioned the search function in Takens and SingleTakens).